code/modules/vore/fluffstuff/guns/gunsword.dm

```diff
@@ -1,6 +1,6 @@
 /obj/item/weapon/gun/energy/gun/fluff/gunsword
-	name = "Sword Buster"
-	desc = "A gun custom built using the science behind a Golden Empire pistol."
+	name = "top-loaded energy pistol"
+	desc = "The Sword Buster gun is custom built using the science behind a Golden Empire pistol. The cell can be removed in close range and used as energy shortsword."
```

Comment:
Oh! This is Kisuke's gun, isn't it?

Reply:
I have no idea, I just spawned it and you can use the energy cell to stab people.
It's pretty neat.